package.json

```diff
@@ -32,7 +36,7 @@
     "@angular/platform-browser-dynamic": "^9.0.1",
     "@angular/router": "^9.0.1",
     "@ng-bootstrap/ng-bootstrap": "^5.3.0",
-    "@wvr/elements": "0.0.2",
+    "@wvr/elements": "0.0.4-rc",
```

Comment:
There was an issue with the publication process with 0.0.4 so we will be moving forward with 0.0.4-rc which was an identical build.